src/components/list/list.less

```
@@ -46,6 +46,7 @@
   padding-left: var(--padding-left);
   position: relative;
   background-color: #ffffff;
+  line-height: normal;
```

我找了下设计稿核对了一下，还是改成固定 1.5 好了

```suggestion
  line-height: 1.5;
```